app/api/answers/get.go

```diff
@@ -36,3 +64,39 @@ func (srv *Service) get(rw http.ResponseWriter, httpReq *http.Request) service.A
 	render.Respond(rw, httpReq, convertedResult)
 	return service.NoError
 }
+
+// OK. Normal response of the itemAnswersGet service
+// swagger:response itemAnswerGetResponse
+type itemAnswerGetResponse struct { //nolint:unused,deadcode
```

Comment:
Why doesn't any linter complain about the fact the comment does not start with "itemAnswerGetResponse"? 
When unused, an alternative (better or not?) would be to directly put it in the service comment directly (not through refs).  Cfr what I have written in the comments on top of `doc/errors.go`.
Strange to have an extra import for something unused... 

(open question, may require to discuss it)

Reply:
> Why doesn't any linter complain about the fact the comment does not start with "itemAnswerGetResponse"?

Probably it is because itemAnswerGetResponse is unexported

Reply:
moved itemAnswerGetResponse into app/doc/responses.go to eliminate extra imports for the unused code